src/build123d/topology.py

```diff
@@ -3246,10 +3198,12 @@ class Comparable(metaclass=ABCMeta):
     """Abstract base class that requires comparison methods"""
 
     @abstractmethod
-    def __lt__(self, other: Any) -> bool: ...
+    def __lt__(self, other: Any) -> bool:
+        ...
```

Comment:
I don't know what formatting you ran but it might be good to run the same as the source repo so this kind of changes doesn't appear.

Reply:
Make `geom_type` a property would be nice aswell

Reply:
I ran ruff, aka fast black. I didn't see anything about a formatter, only pylint and mypy

Reply:
I think Roger uses black. Maybe setting a standard if there isn't one could be great to avoid formatting changes !

Reply:
I use black.  Not sure how the `...` happened but don't worry about it.

Reply:
The property change is good.  There are a few other places where this needs to be done (e.g. `is_valid` IIRC).